docs/en/Configuring-Alluxio-with-HDFS.md

```diff
@@ -45,11 +44,6 @@ Please visit the
 [Building Alluxio Master Branch](Building-Alluxio-Master-Branch.html#distro-support) page for more
 information about support for other distributions.
 
-If everything succeeds, you should see
```

Comment:
do users have a way to know where to find compiled client and server jars?

Reply:
IMO, this is some information that is unnecessary for user who wants to follow this doc.
I think this level of details should be moved to https://www.alluxio.org/docs/master/en/Building-Alluxio-Master-Branch.html  I could update it.

Reply:
nvm, I was thinking about using hadoop as a compute framework where the user needs access to the client jar. For an underfs it's fine since the user only needs to interact with scripts in `bin` which already know where the server jar lives